src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLabeledBreak.java

```diff
@@ -0,0 +1,77 @@
+/*
+FallThrough
+checkLastCaseGroup = (default)false
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThroughLabeledBreak {
+
+    int method(int a, int b) {
+        top: switch (a) {
+        case 1:
+            switch (b) {
+            case 10:
+                break top; // ok
+            default:
+                return 11;
+            }
+        case 2:
+            return 2;
+        }
+
+        outer: outer2: switch (a) {
+            case 1:
+                inner1: inner2: switch (b) {
+                case 10:
+                    break outer; // ok
+                default:
+                    return 11;
+                }
+            case 2:
+                inner: for (;;) {
```

Comment:
Make a similar case like this but have the for loop have a condition.

Also similar with a while loop like:
````
		case 1:
            while(cond) {
                break top;
            }
		case 2: // line 11
````
or point me to where I can find them.

Reply:
I added examples for while loop.

Regarding for cycle - I dont see what difference condition makes, I made infinite only to make it shorter. I brought condition back

Reply:
@strkkk I don't have an issue with this PR, or the issue, or the shorthand loops which look like infinity.

>  I brought condition back

Please have both if you removed one. I just wanted to see how fall through behaves in the different situations with your changes.

The while loop I asked for is just me seeing something that @romani said in the main issue I wanted confirm.

====

My thought process:

https://github.com/checkstyle/checkstyle/issues/11230#issuecomment-1474870614
> nested switch shoudl not be something special

https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java#L390
Sees this code and wonders why it isn't `true, true` as it seems that would also fix the issue. What fails if I turned this into `true, true`.

https://github.com/checkstyle/checkstyle/blob/ea919d5434653046e133aa7ea2f5b410d5a2c683/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough3.java#L74-L77
This violation on 78 goes away which is what I asked to be created.

I just wanted to see how your code behaves when it is a `break <label>` instead of just a break.

Playing in eclipse made me believe the condition changes making it drop through or not. Thats why I asked for a similar case and then one adding a condition.